yaks/lib/yaks/default_policy.rb

```diff
@@ -5,7 +5,7 @@ class DefaultPolicy
     # Default policy options.
     DEFAULTS = {
       rel_template: "rel:{rel}",
-      namespace: Kernel
+      namespace: Object
```

Comment:
There's a reason for it but I don't remember right now. I'll think about it again later and give you an answer.